girder/utility/s3_assetstore_adapter.py

```diff
@@ -392,7 +392,13 @@ def downloadFile(self, file, offset=0, headers=True, endByte=None,
 
         if headers:
             if file['size'] > 0:
-                url = urlFn(key=file['s3Key'])
+                queryParams = {}
+                if contentDisposition == 'inline':
+                    # Tell S3 to set Content-Disposition response header,
+                    # though this only works for non-anonymous connections.
+                    queryParams['response-content-disposition'] = \
+                        'inline; filename="%s"' % file['name']
```

Comment:
I assume S3 just defaults to ``attachment``? A code comment here noting that fact would be good.